src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const sublist = item.querySelector('ul');
+
+  if (sublist) {
+    const header = document.createElement('span');
+    const parentElement = sublist.parentElement;
+
+    header.textContent = parentElement.firstChild.textContent.trim();
```

Comment:
The line `header.textContent = parentElement.firstChild.textContent.trim();` assumes that the first child of `parentElement` is a text node. If the first child is an element node, this will result in an error. Consider checking if the first child is a text node or use a different method to get the desired text content.